x-pack/auditbeat/tests/system/test_metricsets.py

```diff
@@ -21,6 +21,23 @@ def test_metricset_host(self):
         # Metricset is experimental and that generates a warning, TODO: remove later
         self.check_metricset("system", "host", COMMON_FIELDS + fields, warnings_allowed=True)
 
+    @unittest.skipUnless(sys.platform == "linux2", "Only implemented for Linux")
+    def test_metricset_login(self):
```

Comment:
This test should be restricted to little endian architectures (or some other constraint) because `tests/files/wtmp` is from a LE machine.

Reply:
done